### ISSUE TITLE
Cannot read property 'init' of undefined

### ISSUE THREAD

**Issue by sonnyzhou** [tapquo/Lungo.js#350]
here is my code:

``` html
 <script src="js/quo.js"></script>
<script src="js/lungo.js"></script>
<script type="text/javascript">
        Lungo.init({
        name: 'example' 
       });
</script>
```

The console in Chrome got this:
`Cannot read property 'init' of undefined`
and , the background of website is always black?

what's wrong?

**Comment by davidbermu** [comment]
same problem here... :(
I tryed with old versions of Lungo and the webapp started correctly... then, I don't know what is wrong in the latest versions of Lungo

**Comment by sonnyzhou** [comment]
@davidbermu Yeah, I also did it with old version.The Lungo's documentation sucks.
